app/src/main/java/com/willowtree/vocable/CategoriesUseCase.kt

```diff
@@ -0,0 +1,15 @@
+package com.willowtree.vocable
+
+import com.willowtree.vocable.presets.IPresetsRepository
+import com.willowtree.vocable.room.CategoryDto
+import kotlinx.coroutines.flow.Flow
+
+class CategoriesUseCase(
+    private val presetsRepository: IPresetsRepository
+) {
+
+    fun categories(): Flow<List<CategoryDto>> {
```

Comment:
Any reason you aren't just invoking this use case given that there's only one function.
Ref: https://chrynan.codes/invoking-usecases-the-kotlin-way/

```suggestion
    operator fun invoke: Flow<List<CategoryDto>> {
```

Reply:
Yea I've seen that pattern, do you feel strongly that it's more readable that way? I've always felt it's pretty 6-in-one-half-a-dozen-in-the-other, and it gets awkward as soon as you want to add a second function

Reply:
I don't feel strongly about it. Just thought I'd point it out.